contrib/trivy/pkg/converter.go

```diff
@@ -147,9 +147,10 @@ func Convert(results types.Results) (result *models.ScanResult, err error) {
 			libScanner.Type = trivyResult.Type
 			for _, p := range trivyResult.Packages {
 				libScanner.Libs = append(libScanner.Libs, models.Library{
-					Name:     p.Name,
-					Version:  p.Version,
-					FilePath: p.FilePath,
+					Name:       p.Name,
+					Version:    p.Version,
+					PackageURL: p.Identifier.PURL.String(),
```

Comment:
If you are using Trivy version 0.49.0 or earlier, wouldn't a panic occur at this point because Identifier.PURL does not exist?

Reply:
That's correct. I need to fix that bug.